packages/core/parcel-bundler/src/assets/SASSAsset.js

```diff
@@ -45,6 +45,7 @@ class SASSAsset extends Asset {
       ? opts.importer
       : [opts.importer];
     opts.importer.push((url, prev, done) => {
+      url = url.replace('file://', '');
```

Comment:
probably should only replace it if it is at the beginning

Reply:
Updated :)